src/main/java/dk/kb/present/DSCollection.java

```diff
@@ -99,17 +99,13 @@ public class DSCollection {
     public DSCollection(YAML conf, StorageHandler storageHandler) {
         id = conf.keySet().stream().findFirst().orElseThrow();
         try {
-            origin = conf.getSubMap(id).getString("origin");
             conf = conf.getSubMap(id); // There must be some properties for a storage
+            origin = conf.getString("origin");
             prefix = conf.getString(PREFIX_KEY);
             description = conf.getString(DESCRIPTION_KEY, null);
+            recordType = RecordTypeDto.valueOf(conf.getString(RECORDTYPE_KEY));
```

Comment:
This makes the retrieval type mandatory. This makes 100% sense for transforming views, but if `StorageRecords-Lines` are requested, wouldn't the caller want all types? Or maybe we should reconsider the `StorageRecords-Lines` option and maybe make it an explicit endpoint for exporting "raw" data?

Reply:
I like the idea of a raw-endpoint